all/guide/sso/index.md

```diff
@@ -1,14 +1,9 @@
----
-sidebar: auto
-sidebarDepth: -1
----
-
 # Single Sign-On (SSO)
 
 ::::: row
 :::: left
 
-Single Sign-On lets users log into your service by verifying their identity with a 3rd party provider instead of providing an email and password.
+xSingle Sign-On lets users log into your service by verifying their identity with a 3rd party provider instead of providing an email and password.
```

Comment:
Is this a typo or intentional?

Reply:
That's a typo.
```suggestion
Single Sign-On lets users log into your service by verifying their identity with a 3rd party provider instead of providing an email and password.
```